manifests/1.2.0/opensearch-dashboards-1.2.0.yml

```diff
@@ -22,4 +22,7 @@ components:
 - name: indexManagementDashboards
   repository: https://github.com/opensearch-project/index-management-dashboards-plugin
   ref: "main"
+- name: anomalyDetectionDashboards
+  repository: https://github.com/opensearch-project/anomaly-detection-dashboards-plugin
+  ref: "main"
```

Comment:
From my understanding this should point to `1.2` branch (hasn't been cut yet), instead of `main` which is a moving target. See the [1.1 manifest](https://github.com/opensearch-project/opensearch-build/blob/main/manifests/1.1.0/opensearch-dashboards-1.1.0.yml)

Reply:
That's fine now though, and can be updated when the branch is cut.

Reply:
> That's fine now though, and can be updated when the branch is cut.

Yeah, makes sense. I was thinking to update this after a 1.2 branch was cut to simplify the number of PRs made & reviewed, but if this is needed to unblock infra before that is done, then this seems fine.